src/aznfswatchdog

```diff
@@ -131,13 +131,26 @@ reconcile_conntrack()
 
 vecho "Starting aznfswatchdog..."
 
+# Detect and log distro and bash version
+if [ -f /etc/centos-release ]; then
+    linux_distro=$(cat /etc/centos-release)
```

Comment:
linux_distro=$(cat /etc/centos-release 2>&1)

to direct error to the variable in case there's an error in reading, helps to know that